addons/l10n_in/models/account_invoice.py

```diff
@@ -188,7 +188,7 @@ def _l10n_in_get_hsn_summary_table(self):
             base_lines.append({
                 'l10n_in_hsn_code': line.l10n_in_hsn_code,
                 'quantity': line.quantity,
-                'price_unit': line.price_unit,
+                'price_unit': line.price_unit * (1 - (line.discount / 100.0)),
```

Comment:
checking how it's done in other module, often it seems we do:

```suggestion
                'price_unit': line.price_unit * (1 - (line.discount or 0.0) / 100.0),
```
so I guess that discount could be possibly False or None and we might need to take it into account too